src/frontend/dummy_data/posts.json

```diff
@@ -0,0 +1,36 @@
+      "updated": "October 15, 2019",
+      "published": "October 15, 2019",
+      "url": "https://neilong31.wordpress.com/2019/09/13/web-based-note-taker-forevernote/",
+      "site": "https://neilong31.wordpress.com/",
+      "guid": "null"
```

Comment:
Should be something other than null.  Our guids look like `"tag:blogger.com,1999:blog-3586542501367863430.post-1706207209862339192"`, basically a URI of some type.